src/main/java/frc/robot/Robot.java

```diff
@@ -51,9 +57,16 @@ public void robotInit() {
   @Override
   public void robotPeriodic() {
     CommandScheduler.getInstance().run();
+    if (HALUtil.getFPGAButton()) {
+      isBreak = !isBreak;
+    }
     // RobotContainer.APRILTAGS_LIMELIGHT.periodic();
   }
 
+  public static IdleMode getIsBreak() {
```

Comment:
The name "getIsBreak" suggests the function returns the boolean value of "isBreak"
Better naming would be something like "getIdleMode" or "getBreakeMode" etc.